modules/@apostrophecms/image/ui/apos/components/AposMediaManager.vue

```diff
@@ -318,6 +318,21 @@ export default {
       this.uploading = false;
       await this.getMedia();
 
+      if (Array.isArray(imgIds) && imgIds.length && this.items.length === 0) {
+        const [ widgetOptions = {} ] = apos.area.widgetOptions;
+        const [ width, height ] = widgetOptions.minSize || [];
+        await apos.notify('apostrophe:minSize', {
```

Comment:
The notification message is the same as in modules/@apostrophecms/schema/ui/apos/components/AposInputRelationship.vue line 14? 
Actually, I get this
<img width="275" alt="image" src="https://user-images.githubusercontent.com/11479686/212944031-0e394d49-ea32-4c21-80b5-85afafc416bc.png">

Reply:
It was linked to missing dependencies, probably connect-multipart after a npm link